tests/core/scope_test.py

```diff
@@ -32,7 +32,13 @@ def f(scope):
 
     init(f)(random.PRNGKey(0))
 
-  
+  def test_inconsistent_param_shapes(self):
+    def f(scope):
+      scope.param('test', nn.initializers.ones, (4,))
+    
+    msg = 'Inconisist shapes between value and initializer for paramater "test": (2,), (4,)'
```

Comment:
"Inconsistent shapes between value and initializer for parameter" - the spelling fix is breaking the test :)